apps/remix-ide/src/app/udapp/run-tab.tsx

```diff
@@ -233,7 +233,13 @@ export class RunTab extends ViewPlugin {
             "symbol": "ETH",
             "decimals": 18
           })
-        await addCustomInjectedProvider(10, event, 'injected-metamask-gnosis', 'Gnosis Mainnet', '', ['https://rpc.ankr.com/gnosis', 'https://1rpc.io/gnosis'],
+        await addCustomInjectedProvider(10, event, 'injected-metamask-gnosis', 'Gnosis Mainnet', '', ['https://gnosis.drpc.org'],
+          {
+            "name": "XDAI",
+            "symbol": "XDAI",
+            "decimals": 18
+          })
+        await addCustomInjectedProvider(11, event, 'injected-metamask-chiado', 'Gnosis Chiado', '', ['https://gnosis-chiado.drpc.org'],
```

Comment:
would it be fine to make network name as `Gnosis Chiado Testnet` to make it easy for users?

Reply:
`Testnet` is removed again